package.json

```diff
@@ -137,13 +137,21 @@
   },
   "lint-staged": {
     "*.@(ts|tsx)": ["tslint --fix", "yarn run prettier-write --", "git add"],
-    "*.json": ["yarn run prettier-write --", "git add"]
+    "*.json": ["yarn run prettier-write --", "git add"],
+    "*.md": ["yarn run prettier-write --", "git add"]
   },
   "config": {
     "react-native-storybook-loader": {
       "searchDir": ["./src"],
       "pattern": "**/*.story.tsx",
       "outputFile": "./storybook/storyLoader.js"
     }
+  },
+  "prettier": {
+    "printWidth": 120,
+    "semi": false,
+    "singleQuote": false,
+    "trailingComma": "es5",
+    "bracketSpacing": true
```

Comment:
I updated prettier, then backtracked on that, so this is there for when we update